TimberWinR/Inputs/UdpInputListener.cs

```diff
@@ -141,6 +146,32 @@ private void ProcessDataLoop()
             Finished();
         }
 
+        //
```

Comment:
Could you an a level of inheritance here to avoid duplicate code in tcp and udp input listeners?